### ISSUE TITLE
`histogram` and `freqpoly` not working

### ISSUE THREAD

**Issue by davidhodge931** [davidhodge931/ggblanket#241]
Was working in version 1.4.0.  Now it's not. 
```
library(tidyverse)
library(ggblanket)

iris %>% 
  gg_histogram(x = Sepal.Length)

iris %>% 
  ggplot() +
  geom_histogram(aes(x = Sepal.Length)) +
  scale_y_continuous(limits = c(0, 12), breaks = seq(0, 12 , 2))

iris %>% 
  gg_histogram(x = Sepal.Length)
```
Also need the limits to affect the layer data

**Comment by davidhodge931** [comment]
```
library(tidyverse)
library(ggblanket)

iris %>% 
  gg_histogram2(x = Sepal.Length)

p <- iris %>% 
  ggplot() +
  geom_histogram(aes(x = Sepal.Length)) 

p

p %>% 
  layer_data() %>% 
  tibble() %>% 
  ggplot() +
  geom_rect(aes(xmin = xmin, xmax = xmax, ymin = ymin, ymax = ymax))
```

**Comment by davidhodge931** [comment]
Fixed v1.6.0
